src/uptane/uptanerepository.h

```diff
@@ -33,7 +33,7 @@ class Repository {
   // void addSecondary(const std::string &ecu_serial, const std::string &hardware_identifier);
   Json::Value updateSecondaries(const std::vector<Uptane::Target> &secondary_targets);
   std::pair<int, std::vector<Uptane::Target> > getTargets();
-  std::string getPrimaryEcuSerial() { return primary_ecu_serial; };
+  std::string getPrimaryEcuSerial() const { return primary_ecu_serial; };
```

Comment:
+const happy

Reply:
I mean, there's no reason not to, right?

Reply:
Absolutely. Adding missing const-ness is always a good thing